ansible-allinone-demo-vm/roles/opencast-allinone/templates/opencast-ingest.sh

```diff
@@ -66,6 +86,38 @@ SERIESID="$(set -ue;
     --data-urlencode "acl=${SERIESACL}" \
   | sed 's_^.*<dcterms:identifier>\(.*\)</dcterms:identifier>.*$_\1_')"
 
+SERIESXML2='<?xml version="1.0"?>
```

Comment:
Since we now have a few more of these, what about using real names:

```suggestion
SERIES_XML_WIKI_COMMONS='<?xml version="1.0"?>
```